lib/jwt/request_authorizer.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module JWT
+  class RequestAuthorizer < Authorizer
+    class << self
+      attr_writer :token_extractor
```

Comment:
Wouldn't be problems with thread safety here?

Reply:
![i](https://media.tenor.com/images/704079750051cfcba32b2645de8027b6/raw)

If one uses the writer responsibly (that is, doesn't use it after class is loaded) there should be no problems. I don't imagine an use case when somebody would want to change it... I'm not sure.